main.tf

```diff
@@ -44,4 +44,18 @@ resource "aws_wafv2_web_acl_logging_configuration" "default" {
       }
     }
   }
+
+
+  logging_filter {
+    default_behavior = var.logging_filter.default_behavior
```

Comment:
@isaccavalcante this needs some work. For starters, this `logging_filter` block will need to be `dynamic` so it is only included if a consumer of this module passes a value for `logging_filter`. Once it's `dynamic`, you'll want to use `for_each` to make it so it only creates content according to the found filter in the value. 

Give it an update and ping me when you've got something 👍